src/js/trove/make-standalone.js

```diff
@@ -1,8 +1,30 @@
 ({
   requires: [],
-  nativeRequires: ["path", "fs", "requirejs"],
+  nativeRequires: ["path", "fs"],
   provides: {},
   theModule: function(runtime, namespace, uri, path, fs, requirejs) {
```

Comment:
Should remove requirejs from this list of arguments since it's no longer required above.